src/components/ActivityFilters/ActivityFilters.tsx

```
@@ -702,10 +695,8 @@ export const ActivityFilters = ({
   );
 };
 
-export interface ActivityFiltersPopoverProps extends Omit<
-  ActivityFiltersProps,
-  'variant' | 'onFiltersChange'
-> {
+export interface ActivityFiltersPopoverProps
+  extends Omit<ActivityFiltersProps, 'variant' | 'onFiltersChange'> {
```

ESLint

[error] Replace `⏎··extends·Omit<ActivityFiltersProps,·'variant'·|·'onFiltersChange'` with `·extends·Omit<⏎··ActivityFiltersProps,⏎··'variant'·|·'onFiltersChange'⏎`

(prettier/prettier)